src/python-bindings/common2/py_handle.cpp

```diff
@@ -1 +1,72 @@
+//
+// We can't statically define Python types from C if Py_LIMITED_API is set.
+//
+// See python-bindings.md for some more-detailed ranting about this file.
+//
+//
 
+static void _handle_dealloc(PyObject * self);
+
+
+template<class T>
+struct DynamicPyType {
+    DynamicPyType(const char * module_dot_typename) {
+        type_spec.name = module_dot_typename;
+    }
+
+    typedef struct {
+        PyObject_HEAD
+        T t;
+        // There's no mechanism by which Python could know to use the
+        // initializer we put here when it allocates space for this object,
+        // so let's not tempt people to make bad assumptions.
```

Comment:
We're pretty sure that Python will just malloc() a hunk o' memory here, so we won't get the initialized values; update this comment.